rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -513,6 +510,7 @@ class RosBag2PlayQosOverrideTestFixture : public RosBag2PlayTestFixture
   std::vector<std::shared_ptr<rosbag2_storage::SerializedBagMessage>> messages_;
 };
 
+using rosbag2_storage::Rosbag2QoS;
```

Comment:
That is a bad practice to use `using` outside of the namespace and just before the test. 
Please remove it.

Reply:
Can I move it inside test body? It is already done in other places in the codebase. Otherwise being explicit every time in expressions such as:
```cpp
Rosbag2QoS{Rosbag2QoS{}.transient_local()};
```
would become quite lengthy.